SportsHub/SportsHub.Application/Services/ArticleService.cs

```diff
@@ -21,5 +23,30 @@ public async Task<IEnumerable<Article>> GetAllAsync()
         {
             return await _unitOfWork.ArticleRepository.GetByTitleAsync(title);
         }
+
+        public async Task<bool> CreateArticleAsync(CreateArticleDTO adminInput)
+        {
+            var articleExists = await GetByTitleAsync(adminInput.Title) != null;
+
+            if (articleExists)
+            {
+                return false;
+            }
+
+            var article = new Article()
+            {
+                Title = adminInput.Title,
+                CategoryId = adminInput.CategoryId,
```

Comment:
add validation if CategoryId exists, please. If not then false

Reply:
Added validation for CategoryId but I am not sure if that's the right approach.

Reply:
And is it better to move the GetCategoryById method into a Category service?

Reply:
you can calls Category repository here